src/Akeneo/Tool/Bundle/VersioningBundle/Resources/config/model/doctrine/Version.orm.yml

```
@@ -3,13 +3,11 @@ Akeneo\Tool\Component\Versioning\Model\Version:
     table: pim_versioning_version
     repositoryClass: Akeneo\Tool\Bundle\VersioningBundle\Doctrine\ORM\VersionRepository
     indexes:
-        resource_name_resource_id_idx:
+        resource_name_resource_id_version_idx:
```

please document it as good practice of the why of this index. Otehrwise, it's pretty impossible to know the consequence of changing/removing an index...

```suggestion
        # this index is useful to get the maximum version for a given entity when doing the purge
        # it allows to do an index-only scan, which improved greatly the performance
        # @see https://use-the-index-luke.com/sql/clustering/index-only-scan-covering-index
        resource_name_resource_id_version_idx:
```